src/components/Theme/ThemeToggle/index.tsx

```
@@ -0,0 +1,32 @@
+"use client";
+
+import { MoonIcon } from "@/icons/MoonIcon";
+import { SunIcon } from "@/icons/SunIcon";
+import React, { useState } from "react";
+import { useTheme } from "next-themes";
+
+type Props = {};
+
+const ThemeToggle = (props: Props) => {
```

ESLint

[error] 'props' is defined but never used.

(@typescript-eslint/no-unused-vars)